src/python/grpcio/grpc/early_adopter/implementations.py

```diff
@@ -193,7 +198,7 @@ def __getattr__(self, attr):
 def stub(
```

Comment:
I think it's probably worth rewording these to express that the number is the maximum number of threads that may be created rather than the number of threads that definitely will be created. The number is (ultimately) passed as ThreadPoolExecutor's "max_workers" argument; ThreadPoolExecutor doesn't have a "num_workers" argument.